htlcswitch/failure.go

```diff
@@ -67,6 +104,16 @@ type ErrorEncrypter interface {
 // encryption and must be treated as such accordingly.
 type SphinxErrorEncrypter struct {
 	*sphinx.OnionErrorEncrypter
+
+	ogPacket *sphinx.OnionPacket
```

Comment:
It's set [here](https://github.com/lightningnetwork/lnd/pull/775/files#diff-93706aed560df47864bbf09673f8f0b2R119), but yes it is unused ATM. This will be used when we write the raw onion to disk instead the plaintext shared secret.